docs/static/releasenotes.asciidoc

```diff
@@ -30,6 +31,78 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-8-1]]
+=== Logstash 7.8.1 Release Notes
+
+==== Performance improvements and notable issues fixed
+
+===== Fixed performance regression during pipeline compilation
+
+This regression was introduced in 7.7.0 and described in the release notes of 7.8.0 ***TODOLINK[HERE]***.
+Introducing the logging of the plugin id caused the id to explicity end up in the generated code during compilation, causing an explosion of generated classes and zero reuse of the cache.
+By changing the assignment of plugin.id to load the value dynamically instead, this has now been fixed. Find more details in https://github.com/elastic/logstash/pull/12038[#12038]
+
+===== Multiple fixes to pipeline shutdown and reloading
+
+7.8.1 brings many fixes that improve the stability of the pipeline lifecycle. These include avoiding overlapping state resolution operations when monitoring was enabled (https://github.com/elastic/logstash/pull/12034[#12034]), ensuring that the persistent queue lock is correctly released if a pipeline fails to start https://github.com/elastic/logstash/pull/12023[#12023], and avoid a known JRuby issue by not spawning a Ruby thread from Java code https://github.com/elastic/logstash/pull/11900[#11900].
+
+===== Changes in the dependencies of Beats/TCP/HTTP Input Plugins
+
+To avoid breaking changes and retain control on the set of ciphers exposed in these plugins the `tcnative` + `boringssl` dependencies have been dropped in favor of using JVM supplied ciphers instead.
+Overall you should see no impact but this may result in fewer ciphers being available if the JCE unlimited strength jurisdiction policy is not installed.
+NOTE: this policy is installed by default on versions of the JDK from u161 onwards
```

Comment:
```suggestion
NOTE: This policy is installed by default on versions of the JDK from u161 onwards.
```

Reply:
If you want this to render with the NOTE style, you need a return before line 53.  Otherwise, the sentence is included with the previous paragraph.